.github/workflows/stale.yml

```diff
@@ -0,0 +1,26 @@
+name: Mark stale issues and pull requests
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Run every day at midnight UTC on Sunday
```

Comment:
This comment doesn't make sense? Does it run every day or just on Sunday's?

(I'd prefer every day at midnight)

Reply:
it means any day (1-31) of any month  (January to December) but it needs to be the Sunday.
I will switch to `0 0 * * *` to run every day at midnight 👍